docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -25,8 +25,16 @@ that doesn't support tree-shaking.
 ## Development environment
 
 Development bundles can contain the full library which can lead to **slower startup times**.
-This is especially noticeable if you import from `@mui/icons-material`.
-Startup times can be approximately 6x slower than without named imports from the top-level API.
+This is especially noticeable if you import using named imports from `@mui/icons-material`.
+Startup times can be approximately 6x slower with named imports than without e.g.
+
+In some cases
+
+`import { Delete } from '@mui/icons-material'`
+
+can be slower than
+
+`import Delete from '@mui/icons-material/Delete'`
```

Comment:
Here's how I would phrase this for a more condensed style, and also to introduce the term "default" as the alternative to a "named" import.

```suggestion
This is especially noticeable if you use named imports from `@mui/icons-material`, which can be up to six times slower than the default import.
For example, between the following two imports, the first (named) will be significantly slower than the second (default):

```
`import { Delete } from '@mui/icons-material'`

`import Delete from '@mui/icons-material/Delete'`
```

Reply:
This looks better. Thanks, @samuelsycamore!
I do have one remark though - it may give the reader an impression that default imports are faster than named ones, but it's not the case. It just happens that we use default exports in the deeper paths (@mui/icons-material/Delete), but if we used named exports there, the performance implications would be the same. I would emphasize the importance of importing from a subpath of a package (the **/Delete** part), instead of named vs. default import.

BTW, we now miss a closing triple tick to end the code block.